src/test/scala/cz/alenkacz/db/postgresscala/PostgresqlAsyncIntegrationTest.scala

```diff
@@ -13,19 +18,107 @@ import org.scalatest.junit.JUnitRunner
 @RunWith(classOf[JUnitRunner])
 class PostgresqlAsyncIntegrationTest extends AsyncFlatSpec with Matchers with TestPostgresConnection {
 
-  "Postgres scala" should "return result for basic query" in {
+  "Postgres scala" should "return result for basic string query" in {
     createTestTable()
-    val expected = TestObject("a", Seq("a", "b"))
-    connection.execute("INSERT INTO abc (a, b) VALUES ('a', '{\"a\", \"b\"}')").flatMap(_ => sql"SELECT a,b FROM abc".query(r => TestObject(r(0).string, r(1).strings))).flatMap(actual => {
-      actual.head should be(expected)
+    connection.execute("INSERT INTO abc (s, sa) VALUES ('a', '{\"a\", \"b\"}')").flatMap(_ => sql"SELECT s,sa FROM abc".query(r => (r(0).string, r(1).strings))).flatMap(actual => {
+      actual.head should be(("a", Seq("a", "b")))
+    })
+  }
+
+  it should "return result for basic int query" in {
+    createTestTable()
```

Comment:
I don't like we have to repeat `createTestTable`. Couldn't it be solved somehow?